packages/accounts/src/light-account/e2e-tests/light-account.test.ts

```diff
@@ -1,19 +1,18 @@
-import {
-  LocalAccountSigner,
-  SmartAccountProvider,
-  type SmartAccountSigner,
-} from "@alchemy/aa-core";
+import { LocalAccountSigner, type SmartAccountSigner } from "@alchemy/aa-core";
 import { isAddress, type Address, type Chain, type Hash } from "viem";
 import { generatePrivateKey } from "viem/accounts";
 import { sepolia } from "viem/chains";
-import { LightSmartContractAccount } from "../account.js";
-import { getDefaultLightAccountFactoryAddress } from "../utils.js";
 import {
   API_KEY,
   LIGHT_ACCOUNT_OWNER_MNEMONIC,
-  RPC_URL,
+  PAYMASTER_POLICY_ID,
   UNDEPLOYED_OWNER_MNEMONIC,
 } from "./constants.js";
+import {
+  LightSmartContractAccount,
+  getDefaultLightAccountFactoryAddress,
+} from "../../index.js";
+import { AlchemyProvider } from "../../../../alchemy/src/provider.js";
```

Comment:
think this should come from `@alchemy/aa-alchemy`

Reply:
I can't because core packages does not have @alchemy/aa-alchemy as dependency. Maybe I can add as dev dependency.